### PR TITLE
Add RWSClient::getIOSignals().

### PR DESCRIPTION
**Pull request by de-vri-es**
Getting a list of all signals is very useful if you're still getting to know the robot. I wanted this functionality for a ROS wrapper we've made at RoboHouse [1]. For now, it needs a forked `abb_librws` with this patch applied.

I was kinda waiting for the other PRs to be merged first, but I suppose I might as well make make this one already.

[1] https://gitlab.com/robohouse/abb_io

### CODE REVIEW

File: include/abb_librws/rws_client.h [hunk 1]
```diff
@@ -357,6 +357,13 @@ class RWSClient : public POCOClient
    */
   RWSResult getConfigurationInstances(const std::string topic, const std::string type);
 
+  /**
+   * \brief A method for retrieving the all available IO signals on the controller.
```
Comment (jontje) [inline include/abb_librws/rws_client.h]:
```suggestion
   * \brief A method for retrieving all available IO signals on the controller.
```

Comment (jontje) [reply include/abb_librws/rws_client.h]: Typo.

Comment (de-vri-es) [reply include/abb_librws/rws_client.h]: Good catch!

**Comment by de-vri-es** [comment]
I fixed the typo.

I actually switched from `RWSInterface` to `RWSClient` for `abb_io` because it allows for better error reporting. I might still get around to adding it to `RWSInterface`, but I'll make an issue to be sure it's not forgotten.

**Comment by de-vri-es** [comment]
Issue opened as #66.

**Pull request merged**